Updated Changelog for stable (#19912)
Ultimaker/Cura: resources/texts/change_log.txt

- Fixed a bug where reloading an updated model would result in not all the duplicates of the model being updated
- Fixed a bug that showed incompatible materials when switching between printers with a different filament diameter. 
- Fixed a bug where tree support in enclosed holes has missing layers
- Made improvements in the G-codePathModify slot for Engine Plugins, contributed by @ThomasRahm
- Fixed a bug that prevented changes to the initial layer diameter from being applied properly, contributed by @ThomasRahm
- Fixed a rounding issue in the engine math, @0x5844

* Bugs resolved since the Beta release
- Spiralize outer contour no longer adds unnecessary seams to the model
- When coasting is enabled the printhead no longer drops down 
- Fixed coasting strategy so it's only applied to outer/inner walls, and not to skin, top-bottom, or infill
- Fixed wiping direction when printing a single wall
- Improved wiping in combination with the scarf seam
- Significantly improved the strategy for sharpest corner - smart hiding seam location combination

> - Fixed a bug where long scarf seams and short scarf seam steps would result in the seam being moved outward

Reviewer: ```suggestion
- Fixed a bug where long scarf seam length and short scarf seam steps would result in the seam being moved outward
```

> - Fixed a bug where seam acceleration and deceleration were not working 

Reviewer: ```suggestion
- Fixed a bug where seam acceleration and deceleration around seam were not working
```

- Various setting improvements for ABS and Nylon for UltiMaker Method series
- Various setting improvements for dual extrusion support for the UltiMaker Method series

> - Implemented a hard limit so the bed temperature cannot be more than 120C on all UltiMaker printers.

Reviewer: ```suggestion
- Implemented a hard limit so the bed temperature cannot be more than 120°C on all UltiMaker printers.
```
